### ISSUE TITLE
Theme is not changing to "Dark"

### ISSUE THREAD

**Issue by akshi-11** [fossasia/accounts.susi.ai#502]
**Actual Behaviour**

When the radio button for **"Dark Theme"** is selected, it doesn't change.

**Expected Behaviour**

The theme should change to a dark one

**Steps to reproduce it**

Go to settings/themes. Select the **"Dark"** theme. The theme won't change.

**Screenshots of the issue**

![dark](https://user-images.githubusercontent.com/31557923/45663650-3c9e6e00-bb25-11e8-847a-755b1e04df64.png)

**Would you like to work on the issue?**

Yes

**Comment by tabesin** [comment]
Does a dark theme even exist?

**Comment by akshi-11** [comment]
There is a button for a dark theme and the theme should change like it does in chat.susi.ai.

**Comment by stealthanthrax** [comment]
@akshi-11 @akshi-11  , changing the theme option in accounts, changes the theme in `chat.susi.ai` as of now.

**Comment by singhrohan62** [comment]
@stealthanthrax  I completely agree with you but there is a slight problem.... if I want to change the theme from chat settings, the theme doesn't change. You can give it a try by changing themes from both accounts.susi.ai and chats.susi.ai (the former one works, but not the latter)

**Comment by singhrohan62** [comment]
One more thing, @akshi-11 its just a suggestion for you, disable (don't display) the 'edit theme' button when user opts for light or dark theme because the props for those themes are pre-defined and the button has no use for both those themes.

**Comment by akshi-11** [comment]
@singhrohan62 yes I will do that. But can u please explain why shouldn't the theme in accounts change?

**Comment by singhrohan62** [comment]
@akshi-11 my concern was regarding the problem which I have stated in my first comment.  Btw, I am also in the favour of adding the dark theme in accounts.susi.ai 👍

**Comment by akshi-11** [comment]
@singhrohan62 thank you for the suggestion, I will work on the issue.
